gateway/1.4/modules/ROOT/pages/flex-conn-manage-jenkins.adoc

```diff
@@ -5,52 +5,47 @@ endif::[]
+Automating your Flex Gateway in Connected Mode work flow shares many similarities with the xref:flex-conn-manage-public-api.adoc[Manage Flex Gateway Using the API Manager API] tutorial. Using this Jenkins Pipeline tutorial, you can automate the calls made to the API Manager API. To add additional features from the  https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/api-manager-api/[API Manager API], use the <<create-an-api-instance, Create an API Instance>> and <<apply-an-api-policy, Apply an API Policy>> Pipeline stages as examples.
 
-This tutorial has different steps with different code blocks. As you work through the steps, add the code blocks to the same Jenkins configuration file. Before you can create and deploy an API Instance to your Flex Gateway, you must complete the following steps:
-
-* <<define-useful-functions, Define Useful Functions>>
-* <<define-useful-parameters, Define Useful Parameters>>
-* <<register-and-run, Register and Run>>
-* <<obtain-an-authorization-token-from-access-management, Obtain an Authorization Token from Access Management>>
-
-After you implement these pipeline stages, you can customize your pipeline by making API Manager API requests.
+This tutorial has different steps with different code blocks. As you work through the steps, add the code blocks to the same Jenkinsfile. 
 
 == Before You Begin
-Before you can deploy a Jenkins pipeline, complete the following tasks:
+Before you can deploy a Jenkins Pipeline, complete the following tasks:
 
-* xref:flex-install.adoc[Install Flex Gateway].
+. xref:flex-install.adoc[Install Flex Gateway].
 
-* xref:service-mesh::obtain-connected-apps-credentials.adoc[Configure a Connected App]:
+. xref:service-mesh::obtain-connected-apps-credentials.adoc[Configure a Connected App]:
 
-** Use the *App acts on its behalf (client credentials)* type and include the following scopes:
+.. Use the *App acts on its behalf (client credentials)* type and include the following scopes:
 
 *** API Manager: *Manage APIs Configuration*, *Manage Policies*, *View Policies*, and *Deploy API Proxies*
-*** Runtime Manager: *Read Servers*
+*** Runtime Manager: *Read Servers* and *Manage Servers*
 *** Exchange: *Exchange Viewer*
-+ 
-You do not have to xref:flex-conn-reg-run-app.adoc[Register and Run Flex Gateway with a Connected App] to use the API Manager API, but you can use the same Connected App if you include the proper scopes.
+*** General: *View Organization*
 
-** Save the *Id* and *Secret* of the Connected app you configure.
+.. Save the *Id* and *Secret* of the Connected app you configure.
 
 
-* Collect the Group ID, Asset ID, and Asset Version (GAV) of the Exchange asset you want to create or apply.
+. Collect the Group ID, Asset ID, and Asset Version (GAV) of the Exchange asset you want to create or apply.
```

Comment:
When you talk about fields generically (without referring to specific UI fields), you don't need to capitalize them. So in this case, I would probably use "group ID, asset ID, and asset version). This is throughout the doc. It's up to you to judge whether you are referring to generic things or to specific fields and go from there.